src/lib/dialog/dialog-config.ts

```diff
@@ -57,6 +57,26 @@ export class MatDialogConfig {
   /** Height of the dialog. */
   height?: string = '';
 
+  /** Min-width of the dialog. */
+  minWidth?: string = '';
+
+  /**
+   * Min-height of the dialog.
+   * For this value to be effectively applied, `height` may also need to be defined.
+   * See https://www.w3.org/TR/CSS2/visudet.html#min-max-widths
+   */
+  minHeight?: string = '';
```

Comment:
These don't need to be initialized.

Reply:
Addressed. What are your thoughts on initializing `maxWidth`? I left that initialized for the time being.